galaxy_ng/app/api/ui/serializers/user.py

```diff
@@ -142,11 +144,16 @@ def get_model_permissions(self, obj):
             "add_distribution": obj.has_perm('ansible.add_ansibledistribution'),
             "view_distribution": obj.has_perm('ansible.view_ansibledistribution'),
             "view_user": obj.has_perm('galaxy.view_user'),
-            "delete_user": obj.has_perm('galaxy.delete_user'),
-            "change_user": obj.has_perm('galaxy.change_user'),
-            "add_user": obj.has_perm('galaxy.add_user'),
-            "add_group": obj.has_perm('galaxy.add_group'),
-            "delete_group": obj.has_perm('galaxy.delete_group'),
             "change_group": obj.has_perm('galaxy.change_group'),
             "view_group": obj.has_perm('galaxy.view_group'),
         }
+
+        if not settings.get("SOCIAL_AUTH_KEYCLOAK_KEY"):
```

Comment:
I'd like to set these permissions to `false` in social auth mode, instead of not adding them to the dictionary. They UI might expect these values to be set on the api response.